src/cable/connection.cr

```diff
@@ -125,7 +125,7 @@ module Cable
 
     def unsubscribe(payload : Cable::Payload)
       if channel = Connection::CHANNELS[connection_identifier].delete(payload.identifier)
-        channel.unsubscribed
```

Comment:
`unsubscribed` method acts as a callback that must be called after channel is closed/unsubscribe, so in my mind we must do:

```crystal
channel.close          # we close the connection
channel.unsubscribed   # we call the method giving the opportunity to the channel take some action
```